SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -17,9 +18,18 @@ To see if there were any error during the data import, you can check the **Info*
 
 If the error is handled by the Power BI Connector, the pop-up will contain the error code and error message what you can find in [Errors raised by the Power BI Connector] section to learn more about the error.
 
+#### An error occurs when connecting via the Power BI Service.
+
+If an error occurs when a new connection is created, Power BI Service (Service) will show the exact error message.
+
+If an error oocurs when an existing dataset is refreshed through an existing connection, the _Refresh history_ of that dataset will contain the error details.
+
 ### Creating a technical case
 
-If you cannot resolve the error, reproduce it with trace logging enabled, collect the Power BI mashup trace logs, as described in the [Collecting Power BI mashup trace logs] documentation, and create a [One Identity Technical Case]. Attach the collected trace logs for debugging purposes. Make sure you include the version of your Power BI desktop application, the Power BI Connector and SPS appliance. Attaching an SPS debug bundle may also help us troubleshooting the issue if it is rather related to SPS.
+If you cannot resolve the error, reproduce it with trace logging enabled, collect the Power BI mashup trace logs, as described in the [Collecting Power BI mashup trace logs] documentation. If the error occurs in the Service, collect the Power BI Gateway logs, as described in the [Collecting Power BI Gateway logs] documentation.

```

Comment:
Re: [Collecting Power BI Gateway logs] documentation

I see no content about this in the Standalone Guide. Do we need any?  :-)
(If we did not put info in there on purpose, that's OK, just checking. :-) )

Reply:
No info needed in Standalone Guide, then, I suppose.